packages/xo-server/src/api/xosan.js

```diff
@@ -1132,7 +1132,7 @@ async function _prepareGlusterVm(
     name_description: 'Xosan VM storage',
     memory: memorySize,
   })
-  await xapi.call('VM.set_xenstore_data', newVM.$ref, xenstoreData)
+  await newVM.set_xenstore_data(xenstoreData)
```

Comment:
Why did you choose `Snake case` ? I've never seen this in previous code

Reply:
Because it's automatically generated from the field names which are also snake_case.

This is because it's the same as the Xen-API (e.G. [VM class](https://xapi-project.github.io/xen-api/classes/vm.html)).